### ISSUE TITLE
Define an optional rake or generator to set up rom_factory

### ISSUE THREAD

**Issue by ghost** [janjiss/rom_factory#6]
It will be awesome to have a rake or generator task that sets the rom_factory basic configuration, like:

Create -> ../config/initializers/rom_factory.rb
Inside it, for example Hanami setup:

RomFactory::Config.configure do |config|
  config.container = Hanami::Repository.container
end

Create -> ../spec/rom_factories/

Command line, something like: 

% rake rom_factory:install hanami(whatever framework)
% hanami(whatever framework) generator rom_factory

**Comment by janjiss** [comment]
Good point. I need to think about this a little bit, because we need to figure out a clear way to support different sets of frameworks and potential initialisation locations. For example, I'm not using Rails, so some of these locations might not make sense for me. I'll get back to you on this one.
